### ISSUE TITLE
Clean up warning

### ISSUE THREAD

**Issue by BadChoice** [MatthewYork/DateTools#267]
There is a warning that appears in the last xcode version in `DTTimePeriodChain.m` at line 149

The solution is easy, replace the line

```
[((DTTimePeriod *) obj) shiftEarlierWithSize:DTTimePeriodSizeSecond amount:[periods[0] durationInSeconds]];
```

for 

```
[((DTTimePeriod *) obj) shiftEarlierWithSize:DTTimePeriodSizeSecond amount:[self->periods[0] durationInSeconds]];
```

Let me know if you want a PR for this

**Comment by jnavarrom** [comment]
The PR #266 fixes this warning

**Comment by BadChoice** [comment]
Great! thank you!

**Comment by MatthewYork** [comment]
Fixed in release 5.0.0
